src/main/java/com/mixpanel/android/mpmetrics/InAppNotification.java

```diff
@@ -200,6 +200,6 @@ public InAppNotification createFromParcel(Parcel source) {
     private final String mCallToAction;
     private final String mCallToActionUrl;
 
-    private static final String LOGTAG = "MixpanelAPI.InAppNotification";
+    private static final String LOGTAG = "MixpanelAPI.InAppNotif";
```

Comment:
There is a size limit to LOGTAGS that we've been ignoring, so I've been shortening the tags as I see the errors. (this is issue #176 )